kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtWifiChannelFrequency.java

```diff
@@ -0,0 +1,34 @@
+package org.eclipse.kura.web.shared.model;
```

Comment:
A copyright header is required for all the code contributions

Reply:
OK.